config/checkstyle_checks.xml

```diff
@@ -365,6 +365,9 @@
       <property name="max" value="3"/>
     </module>
     <module name="NestedTryDepth"/>
+    <module name="NoArrayTrailingComma">
+      <property name="severity" value="ignore"/>
```

Comment:
add above property `<!-- This Check is conflicting with ArrayTrailingComma -->`

Reply:
Done